chemicaltoolbox/rdkit/rdkit_descriptors.py

```diff
@@ -1,76 +1,76 @@
 #!/usr/bin/env python
 
-from rdkit.Chem import Descriptors
-from rdkit import Chem
-import sys, os, re
 import argparse
 import inspect
+import sys
+
+from rdkit import Chem
+from rdkit.Chem import Descriptors
+from rdkit_util import get_supplier
```

Comment:
We still have a problem with pulsar that those extra files (get_supplier.py) are not moved to the pulsar endpoint if they are not specified/called in the XML file.

So either you include (touch $__tool_directory__/get_supplier.py) in the command section or you leave this function in this main file.

Reply:
Ah, good to know